contracts/lib/TypedSignature.sol

```diff
@@ -31,13 +31,11 @@ pragma experimental "v0.5.0";
  */
 library TypedSignature {
 
-    enum SignatureType {
-        INVALID,
-        ECRECOVER_NULL,
-        ECRECOVER_DEC,
-        ECRECOVER_HEX,
-        UNSUPPORTED
-    }
+    // Solidity does not offer guarantees about enum values, so we define them explicitly
```

Comment:
Is this true?? If so we should also update the ones in `MarginAdmin`

Reply:
/ anywhere else we use enums publicly

Reply:
will open different PR